docs/machine-learning/how-to-guides/how-to-use-the-automl-api.md

```diff
@@ -12,6 +12,15 @@ Automated machine learning (AutoML) automates the process of applying machine le
 > [!NOTE]
 > This topic refers to the automated machine learning API for ML.NET, which is currently in preview. Material may be subject to change.
 
+## AutoML Implementation
+
+AutoML has collaborated with Microsoft Research teams [Neural Network Intelligence](https://github.com/Microsoft/nni) (NNI) and [Fast and Lightweight AutoML](https://github.com/Microsoft/flaml) (FLAML). These teams help provide additional benefits to the AutoML implementation. These benefits include:
```

Comment:
AutoML is the feature, so I would re-word this to something like:

"The ML.NET team is collaborating with Microsoft Research Teams NNI and FLAML to improve the AutoML experience in ML.NET"